src/createServer.js

```diff
@@ -1,9 +1,43 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
   /* Write your code here */
   // Return instance of http.Server class
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+    res.statusCode = 200;
+
+    res.on('error', (error) => {
+      console.error(error);
+      res.statusCode = 400;
+      res.end();
```

Comment:
Setting the status code to 400 after an error occurs and then ending the response is not effective because the response is already being ended. Consider handling the error before setting the status code and ending the response.